src/scss/includes/search_form.scss

```diff
@@ -314,6 +317,10 @@ input[type="search"] {
     background-size: 34px 34px;
   }
 
+  .main_field_return {
+    display: none;
```

Comment:
This rule should be outside of the media query, as here it breaks the desktop design.
![Capture d’écran de 2020-05-22 11-28-32](https://user-images.githubusercontent.com/243653/82653592-bdc5d680-9c1f-11ea-9937-d28e441613b9.png)